scp/helptopics.php

```diff
@@ -94,12 +94,17 @@
                         }
                         break;
                     case 'disable':
+                        $num=1;
```

Comment:
I feel like there's probably a better way to handle this, I just get weird with counters sometimes. I'm open to suggestions for improvement on the mass process bit.